core/src/test/java/org/opensearch/sql/config/TestConfig.java

```diff
@@ -63,6 +63,16 @@ protected StorageEngine storageEngine() {
       @Override
       public Table getTable(String name) {
         return new Table() {
+          @Override
+          public boolean isExist() {
```

Comment:
```suggestion
          public boolean exists() {
```
would be more conventional in Java.